src/workerd/api/actor-state.c++

```diff
@@ -939,6 +939,40 @@ kj::Maybe<kj::Date> DurableObjectState::getWebSocketAutoResponseTimestamp(jsg::R
   return ws->getAutoResponseTimestamp();
 }
 
+void DurableObjectState::setHibernatableWebSocketEventTimeout(kj::Maybe<uint32_t> timeout) {
```

Comment:
Maybe a dumb question but should we do an upper bound check? As is we are risking overflowing down to 0, which can unset the timeout.

Reply:
I'd guess so. Just not sure what would be a reasonable value here.

Reply:
Assuming a weekly runtime update schedule, `604800` seconds is probably max useful value. Anything larger you might as well just not set a time out at all.